coap-gateway/service/message/response.go

```diff
@@ -25,7 +23,7 @@ func GetResponse(ctx context.Context, messagePool *pool.Pool, code codes.Code, t
 	}
 }
 
-func isTempError(err error) bool {
+func IsTempError(err error) bool {
 	switch {
 	case strings.Contains(err.Error(), "connect: connection refused"),
```

Comment:
Maybe a TODO for the future: We could optimize this by using error.Is to avoid string comparison.